src/server/game/Groups/GroupMgr.cpp

```diff
@@ -92,11 +92,11 @@ void GroupMgr::LoadGroups()
 
         // Delete all groups whose leader does not exist
         CharacterDatabase.DirectExecute("DELETE FROM `groups` WHERE leaderGuid NOT IN (SELECT guid FROM characters)");
+
         // Delete all groups with less than 2 members (or less than 1 for lfg groups)
-        // TODO: This query is not allowed in MySQL 8.0
-        CharacterDatabase.DirectExecute("DELETE `groups` FROM `groups` LEFT JOIN ((SELECT guid, count(*) as cnt FROM group_member GROUP BY guid) t) ON groups.guid = t.guid WHERE t.guid IS NULL OR (t.cnt<=1 AND groups.groupType <> 12)");
+        CharacterDatabase.DirectExecute("DELETE FROM `groups` WHERE guid NOT IN (SELECT guid FROM group_member GROUP BY guid HAVING COUNT(guid) > 1)");
```

Comment:
Are you sure this should be > 1 (greater than one) and not <= 1 (Less or equal to one)?